pkg/mlnet/mlnet.symbols.nupkgproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <Content Remove="$(PackageAssetsPath)$(PackageIdFolderName)\tools\**\*"/>
```

Comment:
It may be nice to add a comment here. But don't re-spin your PR just for the comment. Can add it later.